Proton/Sources/Core/PREditorContentName.m

```diff
@@ -32,8 +32,15 @@ - (instancetype)initWithRawValue:(NSString *)rawValue {
 }
 
 - (BOOL)isEqual:(id)other {
-    EditorContentName *otherName = (EditorContentName *)other;
```

Comment:
Theoretically, you can have heterogeneous objects in a collection so this force cast here is wrong.

Reply:
Brilliant. I forgot to ask about this... I didn't know what I was doing. 

Now I know these 9 lines are equivalent to `guard let`.